config_sct_run_batch.yml

```diff
@@ -0,0 +1,9 @@
+# config file for sct_run_batch for run_all.py script
```

Comment:
you should probably add a `-config config_sct_run_batch.yml` on the example call to `sct_run_batch` in the README file